abdulfatai360/politico
# #163620731 Implement GET /parties Endpoint

#### What does this PR do?
Have Read operation work on parties endpoint

#### Description of Task to be completed?
Have ```GET /api/v1/parties``` to work
Response spec:<code><pre>{
    "status": Integer,
    "data": [{
        "id": Integer,
        "name": String
    }]
}
</code></pre>
#### How should this be manually tested?
Clone the repo, ```cd``` into it
- On terminal
Run``` npm test```
Then all tests should pass
- On Postman
Run ```npm start```
Launch Postman and send request
#### What are the relevant pivotal tracker stories?
#163620731
